examples/domainmodel/test/domainmodel-cli.test.ts

```diff
@@ -0,0 +1,52 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import path from 'path';
+import fs from 'fs';
+import { exec, ExecException } from 'child_process';
+
+describe("Test the domainmodel CLI", () => {
+    let fullPath: string;
+    let fileName = path.join(__dirname, '../example/datatypes.dmodel');
+    const destination = 'domainmodel-example-test';
+    
+    test('Generator command returns code 0', async () => {
+        const result = await cli(['generate', fileName, `-d ${destination}`]);
+        expect(result.code).toBe(0);
+    });
+
+    test('Generated directory exists', () => {
```

Comment:
One test should not rely on another one to run. Merge the two tests to avoid this.

Reply:
I merged the tests and added an Expect-Message package for jest. Unfortunately it just shows messages for failed tests. Anyway, I've left it because it does not hurt either.